packages/block-editor/src/components/rich-text/style.scss

```diff
@@ -53,12 +53,27 @@ figcaption.block-editor-rich-text__editable [data-rich-text-placeholder]::before
 	z-index: z-index(".components-popover.block-editor-rich-text__inline-format-toolbar");
 
 	.components-popover__content {
+		width: auto;
 		min-width: auto;
-		margin-bottom: 6px;
+		margin-bottom: $grid-unit;
+		box-shadow: none;
+
+		// Block UI appearance.
+		border: $border-width solid $dark-gray-primary;
+		border-radius: $radius-block-ui;
+		background-color: $white;
 	}
 
 	.components-toolbar {
 		// The popover already provides a border.
 		border: none;
 	}
+
+	.components-toolbar__control,
+	.components-dropdown-menu__toggle {
+		min-width: $grid-unit-60;
+		min-height: $grid-unit-60;
+		padding-left: $grid-unit-15;
+		padding-right: $grid-unit-15;
+	}
```

Comment:
The duplication of the styles here, makes me wonder if we need a reusable `BlockToolbarWrapper` component or something like that  that applies the styles for inline toolbars and contextual ones similarly. cc @ellatrix

Reply:
Same in https://github.com/WordPress/gutenberg/pull/19344/commits/e522228fa24ec8773fd5b784d3c68673ad2f783a#diff-25b3822e5d2d07ce0d1fe5c87bab489eR49, honestly — right now the block toolbar and the caption toolbar are farther apart than they need to be.